manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/processor/PluginProcessor.java

```diff
@@ -140,14 +146,40 @@ private String applyCustomSnippets(Multimap<String,String> kindVsCustomSnippets,
         return yamlString;
     }
 
-    @SuppressWarnings("java:S1135")
-    private void warningMessageForCustomSnippets(Multimap<String,String> kindVsCustomSnippets){
-        //TODO Support all new resource kinds
-        if(kindVsCustomSnippets != null && !kindVsCustomSnippets.isEmpty()){
-            Set<String> kinds = kindVsCustomSnippets.keySet();
-            String ignoreKindsForCustomSnippets = String.join(",", kinds);
-            WorkflowLogger.warn(ManifestGeneratorActivity.IGNORING_CUSTOM_SNIPPET,ignoreKindsForCustomSnippets);
+    private List<Manifest> getManifestsFromCustomSnippets(ServiceMetadata serviceMetaData, String manifestDir, Multimap<String,String> kindVsCustomSnippets){
+        if(kindVsCustomSnippets == null || kindVsCustomSnippets.isEmpty()){
+            return Collections.emptyList();
+        }
+        List<Manifest> manifestList = new ArrayList<>();
+        kindVsCustomSnippets.forEach((kind,customSnippet)->{
+            WorkflowLogger.startActivity(ManifestGeneratorActivity.GENERATING_MANIFEST, kind);
+            ManifestMeta manifestMeta = new ManifestMeta(kind);
```

Comment:
manifestFileGenerator.getYamlManifest() is internally using this to creating resource file name. handle ManifestMeta.identifier?